drivers/modem/modem_cmd_handler.c

```diff
@@ -510,7 +511,8 @@ int modem_cmd_send_nolock(struct modem_iface *iface,
 			  struct modem_cmd_handler *handler,
 			  struct modem_cmd *handler_cmds,
 			  size_t handler_cmds_len,
-			  const u8_t *buf, struct k_sem *sem, int timeout)
+			  const u8_t *buf, struct k_sem *sem,
```

Comment:
It'd be nice to avoid formating changes in commits that make non-whitespace changes.

Reply:
> Change requested for the west manifest update in a commit that doesn't mention why it's changing. The rest is commentary.

Thanks for the review! I will ping you once I've pushed an update.